operations/utility_storage/Troubleshoot_S3FS_Mounts.md

```diff
@@ -84,3 +84,71 @@ ncn-mw: # mount -a
 
 If the above command fails, then the error likely indicates that there is an issue communicating with Ceph's `Radosgw` endpoint (`rgw-vip`).
 In this case the [Troubleshoot an Unresponsive S3 Endpoint](Troubleshoot_an_Unresponsive_S3_Endpoint.md) procedure should be followed to ensure the endpoint is healthy.
+
+## Mount 'admin-tools' S3 bucket
+
+In CSM 1.2 and CSM 1.3, `/var/lib/admin-tools (admin-tools S3 bucket)` was a mounted S3 bucket. Starting in CSM 1.4, the `admin-tools` S3 bucket is no longer mounted.
+It is not necessary for this bucket to be mounted for system operations.
+However, if it is desired to have the `admin-tools` S3 bucket mounted, please follow the steps below.
+
+  1. (`ncn-m#`) Mount the `admin-tools` S3 bucket. Run this on each master node where the bucket should be mounted.
+
+      ```bash
+      function mount_admin_tools() {
+        s3_bucket="admin-tools"
+        s3fs_mount_dir="/var/lib/admin-tools"
+        s3_user="admin-tools"
+        
+        s3fs_cache_dir=/var/lib/s3fs_cache
+        if [ -d ${s3fs_cache_dir} ]; then
+          s3fs_opts="use_path_request_style,use_cache=${s3fs_cache_dir},check_cache_dir_exist,use_xattr"
+        else
+          s3fs_opts="use_path_request_style,use_xattr"
+        fi
+
+        echo "Configuring for ${s3_bucket} S3 bucket at ${s3fs_mount_dir} for ${s3_user} S3 user"
+
+        mkdir -p ${s3fs_mount_dir}
+
+        pwd_file=/root/.${s3_user}.s3fs
+        access_key=$(kubectl get secret ${s3_user}-s3-credentials -o json | jq -r '.data.access_key' | base64 -d)
+        secret_key=$(kubectl get secret ${s3_user}-s3-credentials -o json | jq -r '.data.secret_key' | base64 -d)
+        s3_endpoint=$(kubectl get secret ${s3_user}-s3-credentials -o json | jq -r '.data.http_s3_endpoint' | base64 -d)
+
+        echo "${access_key}:${secret_key}" > ${pwd_file}
+        chmod 600 ${pwd_file}
+
+        echo "Mounting bucket: ${s3_bucket} at ${s3fs_mount_dir}"
+        s3fs ${s3_bucket} ${s3fs_mount_dir} -o passwd_file=${pwd_file},url=${s3_endpoint},${s3fs_opts}
+
+        echo "Adding fstab entry for ${s3_bucket} S3 bucket at ${s3fs_mount_dir} for ${s3_user} S3 user"
+        echo "${s3_bucket} ${s3fs_mount_dir} fuse.s3fs _netdev,allow_other,passwd_file=${pwd_file},url=${s3_endpoint},${s3fs_opts} 0 0" >> /etc/fstab
+
+        echo "Set cache pruning for admin tools to 5G of the 200G volume (every 2nd hour)"
+        echo "0 */2 * * * root /usr/bin/prune-s3fs-cache.sh admin-tools ${s3fs_cache_dir} 5368709120 -silent" > /etc/cron.d/prune-s3fs-admin-tools-cache
+      }
+      mount_admin_tools
```

Comment:
This should be a script in docs-csm, or even libcsm.

Reply:
I have moved this to a script at 'scripts/mount-admin-tools-bucket.sh'. Let me know if you think the script should live somewhere else!